en/flight_controller/pixhawk6c_mini.md

```diff
@@ -0,0 +1,206 @@
+| UART8  | /dev/ttyS6 | GPS2                      | GPS2             |
+
+<!-- See http://docs.px4.io/main/en/hardware/serial_port_mapping.html#serial-port-mapping -->
+
+## Dimensions
+
+![Pixhawk6c Mini Dimensions](../../assets/flight_controller/pixhawk6c_mini/pixhawk_6c_mini_dimension.jpg)
+
+## Voltage Ratings
+
+_Pixhawk 6C Mini_ can be double-redundant on the power supply if two power sources are supplied. The two power rails are: **POWER1** and **USB**.
+
+**Normal Operation Maximum Ratings**
+
+Under these conditions all power sources will be used in this order to power the system:
+
+1. **POWER1** inputs (4.9V to 5.5V)
+1. **USB** input (4.75V to 5.25V)
+
+**Absolute Maximum Ratings**
+
+Under these conditions the system will not draw any power (will not be operational), but will remain intact.
+
+1. **POWER1** inputs (operational range 4.1V to 5.7V, 0V to 10V undamaged)
+1. **USB** input (operational range 4.1V to 5.7V, 0V to 6V undamaged)
+1. Servo input: VDD_SERVO pin of **FMU PWM OUT** and **I/O PWM OUT** (0V to 42V undamaged)
+
+**Voltage monitoring**
+
+Pixhawk 6C Mini uses analog power modules.
+
+Holybro makes various analog [power modules](../power_module/README.md) for different needs:
+
+- [PM02 Power Module](../power_module/holybro_pm02.md)
+- [PM06 Power Module](../power_module/holybro_pm06_pixhawk4mini_power_module.md)
+- [PM07 Power Module](../power_module/holybro_pm07_pixhawk4_power_module.md)
+- [PM08 Power Module](https://holybro.com/products/pm08-power-module-14s-200a)
+
+## Building Firmware
+
+:::tip
+Most users will not need to build this firmware!
+It is pre-built and automatically installed by _QGroundControl_ when appropriate hardware is connected.
+:::
+
+To [build PX4](../dev_setup/building_px4.md) for this target:
+
+```
+make px4_fmu-v6c_default
+```
+
+<a id="debug_port"></a>
+
+## Debug Port
+
+The [PX4 System Console](../debug/system_console.md) and [SWD interface](../debug/swd_debug.md) run on the **FMU Debug** port.
+
+The pinouts and connector comply with the [Pixhawk Debug Mini](../debug/swd_debug.md#pixhawk-debug-mini) interface defined in the [Pixhawk Connector Standard](https://github.com/pixhawk/Pixhawk-Standards/blob/master/DS-009%20Pixhawk%20Connector%20Standard.pdf) interface (JST SH connector).
+
+| Pin     | Signal           | Volt  |
+| ------- | ---------------- | ----- |
+| 1 (red) | `Vtref`          | +3.3V |
+| 2 (blk) | Console TX (OUT) | +3.3V |
+| 3 (blk) | Console RX (IN)  | +3.3V |
+| 4 (blk) | `SWDIO`          | +3.3V |
+| 5 (blk) | `SWCLK`          | +3.3V |
+| 6 (blk) | `GND`            | GND   |
+
+For information about using this port see:
+
+- [SWD Debug Port](../debug/swd_debug.md)
+- [PX4 System Console](../debug/system_console.md) (Note, the FMU console maps to USART3).
+
+## Peripherals
+
+- [Digital Airspeed Sensor](https://holybro.com/products/digital-air-speed-sensor)
+- [Telemetry Radio Modules](https://holybro.com/collections/telemetry-radios?orderby=date)
```

Comment:
It is better for users to link to the telemetry radio page, then link to Holybro recommendations (unless the FC can only work with Holybro hardware).

```suggestion
- [Telemetry Radio Modules](../telemetry/README.md):
  - [Holybro Telemetry Radio](../telemetry/holybro_sik_radio.md)
  - [Holybro Microhard P900 Radio](../telemetry/holybro_microhard_p900_radio.md)
  - [Holybro XBP9X Telemetry Radio](../telemetry/holybro_xbp9x_radio.md)
```